orca-keel/src/main/kotlin/com/netflix/spinnaker/orca/keel/task/ImportDeliveryConfigTask.kt

```diff
@@ -60,9 +64,17 @@ constructor(
         context.repoType, context.projectKey, context.repositorySlug, context.directory, context.manifest, context.ref
       )
 
+      if (context.sendGitInfo) {
```

Comment:
How about just attempting to parse the data and falling back to the default behavior when we can't, instead of using a flag? Similar to what you did on the keel side. One fewer thing to cleanup later...

Reply:
I'm not 100% confident this won't blow up. So I'm going to keep it behind a flag to make sure, and then I'll remove the flag and do the behavior you said. I'd rather have more to clean up but 0 user impact.